.travis.yml

```diff
@@ -6,30 +6,25 @@ cache:
 
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction"
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
     - COVERAGE_DEPS="php-coveralls/php-coveralls"
 
 matrix:
   fast_finish: true
   include:
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=lowest
-    - php: 7.1
+    - php: 7.3
       env:
         - DEPS=latest
-        - CS_CHECK=true
-        - TEST_COVERAGE=true
```

Comment:
- CS_CHECK=true
        - TEST_COVERAGE=true

Should both be added for another version.

Reply:
@geerteltink it's only for php8 or other versions too?
when wee need cs-check and coverage?

Reply:
@geerteltink I think it's safe to keep this on a single CI entry. PHP 8 is too unstable to run these checks there, for now.

Reply:
I know, but in a previous commit it was entirely removed. With a force push you can't see the old commit history.